packages/native/test/lib/ElementAssertion.test.tsx

```diff
@@ -0,0 +1,94 @@
+import { AssertionError, expect } from "@assertive-ts/core";
+import { render } from "@testing-library/react-native";
+import {
+  View,
+  TextInput,
+} from "react-native";
+
+import { ElementAssertion } from "../../src/lib/ElementAssertion";
+
+describe("[Unit] ElementAssertion.test.ts", () => {
+  describe(".toBeDisabled", () => {
+    context("when the element is TextInput", () => {
+      it("returns the assertion instance when is not editable", () => {
+        const element = render(
+            <TextInput testID="id" editable={false} />,
+        );
+        const test = new ElementAssertion(element.getByTestId("id"));
+        expect(test.toBeDisabled()).toBe(test);
+      });
+      it("throws an error when it is editable", () => {
+        const reactElement = render(<TextInput editable={true} testID="id" />);
+        const test = new ElementAssertion(reactElement.getByTestId("id"));
+
+        expect(() => test.toBeDisabled())
+            .toThrowError(AssertionError)
+            .toHaveMessage('Received element <TextInput testID="id"... /> is enabled.');
```

Comment:
Use 2-space indentation:
```suggestion
        expect(() => test.toBeDisabled())
          .toThrowError(AssertionError)
          .toHaveMessage('Received element <TextInput testID="id"... /> is enabled.');
```